test/desktop-lite/test.sh

```diff
@@ -26,7 +26,6 @@ checkOSPackage() {
 
 check "desktop-init-exists" bash -c "ls /usr/local/share/desktop-init.sh"
 check "log-exists" bash -c "ls /tmp/container-init.log"
-check "fluxbox-exists" bash -c "ls -la ~/.fluxbox"
```

Comment:
Should we have this as the default value for `installFluxbox` is `true` ?

Reply:
@samruddhikhandale we can't, because `test.sh` is ran by all autogenerated tests and scenarios.

I have added a scenario where Fluxbox is installed and that check exists, I have also added another scenario where Fluxbox is not installed.

Since there's a scenario where Fluxbox doesn't exist, this check would fail for that scenario.